### PR TITLE
chore: update workflows

### PR DESCRIPTION
**Pull request by misitebao**
- Fix check document workflow
- Add workflow to auto push to crowdin after website update

**Work before the merger**: Add an Actions secret called `CROWDIN_PERSONAL_TOKEN` with the value of your Crowdin [Personal Access Tokens](https://crowdin.com/settings#api-key)

### CODE REVIEW

**Comment by leaanthony** [comment]
Did you mean to delete `build.yml`?

**Comment by misitebao** [comment]
@leaanthony 
Yes, because the `build_and_test.yml` script has it fully covered.

**Comment by leaanthony** [comment]
I was looking at the fact that we are setting the crowdin token in the GITHUB_TOKEN env so went to look at crowdin and found this: https://developer.crowdin.com/github-crowdin-action/ - This seems like the official way of doing this?

**Comment by misitebao** [comment]
@leaanthony 
Both ways can be achieved, and both require setting environment variables, but environment variables cannot be stored in the repository in plaintext, so you can only set `Actions Secret`, and then use it in the workflow file.

**Comment by leaanthony** [comment]
I couldn't find any docs saying that setting GITHUB_TOKEN to the Crowdin token does anything.

File: .github/workflows/push-to-crowdin.yml [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+      - name: Set node
+        if: steps.verify-changed-files.outputs.files_changed != 'true'
+        uses: actions/setup-node@v2
+        with:
+          node-version: 16.x
+
+      - name: Push files
+        env:
+          GITHUB_TOKEN: ${{ secrets.CROWDIN_PERSONAL_TOKEN }}
```
Comment (leaanthony) [inline .github/workflows/push-to-crowdin.yml:33]: I'm trying to understand what this is for. How does the Crowdin cli read this in?

Comment (misitebao) [reply .github/workflows/push-to-crowdin.yml:33]: Oh sorry, the key should be: `CROWDIN_PERSONAL_TOKEN`, my mistake, I will update this branch later.

**Review by leaanthony — APPROVED** [review]

**Pull request merged**